specs/activity-api.yaml

```diff
@@ -406,22 +326,16 @@ paths:
                   type: number
                   format: double
         404:
-          description: Not Found
-          content:
-            application/json:
-              schema:
-                $ref: 'common.yaml#/schemas/ErrorMessage'
+          $ref: 'common.yaml#/responses/NotFound'
         500:
-          description: Server Error
-          content:
-            application/json:
-              schema:
-                $ref: 'common.yaml#/schemas/ErrorMessage'
+          $ref: 'common.yaml#/responses/ServerError'
+
   /activity/{activityId}/command:
+
     get:
       tags:
         - requestor_state
-      summary: Get running command for a specified Activity.
+      summary: Get running commands for a specified Activity.
```

Comment:
This endpoint is unused for the time being; `GetExecBatchResults` is utilized instead. I'm unsure whether extending it to multiple commands has any practical value. Additionally, the endpoint name (`/activity/{activityId}/command`) suggests a single command to be returned.

I'd actually opt for removing this endpoint altogether.

Reply:
Multiple commands may occur when we allow concurrent command execution.
The whole purpose of this particular webmethod is to verify which commands (of a batch) are being executed, implying that previous commands have finished before. This gives a chance to a requestor which has eg. hung or has been restarted. I'm fine with removing this webmethod altogether, as having a persistent GetExecBatchResults() gives better capabilities.

Reply:
Let's settle on removing this particular endpoint then

Reply:
As we're not making GetExecBatchResults() persistent (see conversation below) - I'd like to keep the ability to actively ask about active ExeScript commands.